### DISCUSSION TITLE
Pressing Select in gamelist broke the themes (visual problem, not functional)

### DISCUSSION THREAD

**Discussion by TooGeekCreations** [OnionUI/Onion#997] [Q&A] [answered]
I press SELECT when my game list are visible and the selector pops up in the corner of the screen (launch, add to favorites, delete, search) that one i think it is broken. A little piece off the background image shows up below the launch button and the bg-list-s.png do not covering the whole menu lenght. (bg-pop-menu-1, bg-pop-menu-2, bg-pop-menu-3.png shows without a problem) Also is this menu got a png name so i can change the background of it just like bg-pop-menu-1.png? 
screenshots below:
![MainUI_018](https://github.com/OnionUI/Onion/assets/139160553/4977695d-9971-41e5-adae-deacf72d0761)
![MainUI_019](https://github.com/OnionUI/Onion/assets/139160553/9ed65c32-b789-42d0-9fcd-d92aa6e55319)
![MainUI_020](https://github.com/OnionUI/Onion/assets/139160553/fc8dd856-8e1f-4d5c-ae4c-ceca35f60bfe)
![MainUI_021](https://github.com/OnionUI/Onion/assets/139160553/f67d9e1c-5721-47ee-887d-a8ffc700088e)
![MainUI_022](https://github.com/OnionUI/Onion/assets/139160553/5f5a8455-6577-4ade-9ef0-3f54f9e9add0)
![MainUI_023](https://github.com/OnionUI/Onion/assets/139160553/5a7a98cc-84b5-46f3-a51a-d1a90ec856d9)
![MainUI_024](https://github.com/OnionUI/Onion/assets/139160553/c8726f79-d868-426c-9d7e-5bee0cba7381)
![MainUI_025](https://github.com/OnionUI/Onion/assets/139160553/bf14c28a-90a6-4d83-a235-08b9edddd4cc)

**Comment by Aemiii91** [comment, marked as answer]
Fixed in next release: https://github.com/OnionUI/Onion/pull/938
